buddyboss/buddyboss-platform
# #2121 - activity comment sync fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2121 .

### How to test the changes in this Pull Request:

1. Add a root comment in the Feed section with release branch
2. Add a media/document in the root as well as child also ( Hierarchy comment ) with release branch
3. Now move this branch and check all added comment for media/document
4. FYI - We will not allow user to add like/comment for media/document with this branch. If user has already this type of fees then user can only see comments. Not add new comments for media/document

### Proof Screenshots or Video

https://screencast-o-matic.com/watch/crjviCVQ2jR

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Dependent API
https://github.com/buddyboss/buddyboss-platform-api/pull/314
Dependent App Native
https://github.com/buddyboss/buddyboss-app-native/issues/1565
